test/lib/agent_helper.js

```diff
@@ -205,15 +206,12 @@ const helper = (module.exports = {
     shimmer.unwrapAll()
     shimmer.registeredInstrumentations = Object.create(null)
     shimmer.debug = false
-    // On all versions each agent will add an unhandledRejection handler. This
-    // handler needs to be removed on unload.
-    removeListenerByName(process, 'unhandledRejection', '__NR_unhandledRejectionHandler')
```

Comment:
Further proof that github hides commits really well: https://github.com/newrelic/node-newrelic/pull/1389/commits/2aff3cb208a274fe57325fb1c30bd94376db81b7

Reply:
In the commit message of this commit you can see that there was another commit that removed an event handler for `unhandledRejection`. Since we don't add this event listener anymore and certainly not with the `__NR_unhandleRejectionHandler` name, we don't need this helper function to remove it either.